src/platform/user/authentication/utilities.js

```diff
@@ -79,12 +72,22 @@ function redirectWithGAClientId(redirectUrl) {
   }
 }
 
+function standaloneRedirect() {
+  const { application, to } = window.location.query;
```

Comment:
I believe `location.query` is specific to the react router and not a part of the standard [browser API](https://developer.mozilla.org/en-US/docs/Web/API/Location). I'm going to update this to use `URLSearchParams` along with tests